src/test/components/MarkerChart.test.js

```diff
@@ -135,20 +127,57 @@ it('renders MarkerChart correctly', () => {
   delete window.devicePixelRatio;
 });
 
+it('renders the hoveredItem markers properly', () => {
+  window.devicePixelRatio = 1;
+
+  const profile = getProfileWithMarkers(MARKERS);
+  const {
+    flushRafCalls,
+    dispatch,
+    markerChart,
+    flushDrawLog,
+  } = setupWithProfile(profile);
+
+  dispatch(changeSelectedTab('marker-chart'));
+  markerChart.update();
+  flushRafCalls();
+  flushDrawLog();
+
+  // No tooltip displayed yet
+  expect(markerChart.find('Tooltip').exists()).toEqual(false);
+
+  // Move the mouse on top of an item.
+  markerChart.find('canvas').simulate('mousemove', {
+    nativeEvent: { offsetX: 50, offsetY: 5 },
+    pageX: 50,
+    pageY: 5,
+  });
+  markerChart.update();
+  flushRafCalls();
+
+  const drawCalls = flushDrawLog();
+  expect(drawCalls).toMatchSnapshot();
```

Comment:
I tested this by removing the "fixing" commit, and the snapshot actually changes. Without the commit we clearly see we have the same marker drawn twice (because this marker line has 2 markers).